spec/features/unauthenticated_user/unauth_user_can_login_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.feature "the unauthenticated user", type: :feature do
+  let(:valid_attributes) {
+    {full_name: "Stan The Man",
+      user_name: "Theman",
+      password: "password",
+      role: 0,
+      email: "stan@stan.com"}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.